lib/trailblazer/operation/policy.rb

```diff
@@ -2,6 +2,21 @@
 
 module Trailblazer
   class NotAuthorizedError < RuntimeError
+    attr_reader :query, :record, :policy
+
+    def initialize(options = {})
+      if options.is_a? String
```

Comment:
This is pretty much a copy/paste from pundit... I think generally it's just checking two different options as input... Actually, now that I'm thinking about it I'm not actually sure when the exception passed as the argument here would ever be just a string? But maybe if the Error is being new'd up custom maybe?